static/templates/stream_settings/stream_members.hbs

```diff
@@ -13,6 +13,7 @@
                     <div class="pill-container person_picker">
                         <div class="input" contenteditable="true"
                           data-placeholder="{{t 'Add subscribers. Use @usergroup or #streamname to bulk add subscribers.' }}">
+                            {{~! empty div ~}}
```

Comment:
I'd write a bit more expressive of a comment, e.g. "Squash whitespace so that data-placeholder is displayed"

Reply:
Fixed.